modules/tinymce/src/core/test/ts/browser/focus/KeyboardShortcutElementPathFocusTest.ts

```diff
@@ -0,0 +1,36 @@
+import { FocusTools, Keys } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { SugarDocument } from '@ephox/sugar';
+import { McEditor, TinyContentActions, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import { RawEditorOptions } from 'tinymce/core/api/OptionTypes';
+
+describe('browser.tinymce.themes.silver.editor.toolbar.KeyboardShortcutElementPathFocusTest ', () => {
+
+  const pTestFocus = async (options: RawEditorOptions) => {
+    const editor = await McEditor.pFromSettings<Editor>({
+      menubar: false,
+      ...options,
+      base_url: '/project/tinymce/js/tinymce'
+    });
+
+    const doc = SugarDocument.getDocument();
+
+    TinyContentActions.keystroke(editor, 122, { alt: true });
+
+    await FocusTools.pTryOnSelector('Assert element path is focused', doc, 'div[role=navigation] .tox-statusbar__path-item');
+    TinyUiActions.keystroke(editor, Keys.escape());
+    McEditor.remove(editor);
+  };
+
+  context('TINY-2884: Pressing Alt+F11 focuses the element path and escape from the toolbar will focus the editor', () => {
+    it('classic mode', () =>
+      pTestFocus({ })
+    );
+
+    it('class bottom', () =>
+      pTestFocus({ toolbar_location: 'bottom' })
+    );
+  });
```

Comment:
This isn't how we'd normally do this and `context` blocks should never be associated with a Jira so the it and context are backwards here. Can you please switch it to something like this instead?
```
Arr.each([
  { label: 'classic mode', options: {}},
  { label: 'class bottom', options: { toolbar_location: 'bottom' }}
], (tester) => {
  context(tester.label, () => {
    const hook = TinyHooks.bddSetup({
      menubar: false,
      ...tester.options,
      base_url: '/project/tinymce/js/tinymce'
    });

    it('TINY-2884: Pressing Alt+F11 focuses the element path and escape from the toolbar will focus the editor', async () => {
      const editor = hook.editor();
      const doc = SugarDocument.getDocument();

      TinyContentActions.keystroke(editor, 122, { alt: true });

      await FocusTools.pTryOnSelector('Assert element path is focused', doc, 'div[role=navigation] .tox-statusbar__path-item');
      TinyUiActions.keystroke(editor, Keys.escape());
    });
  });
});
```